package.json

```diff
@@ -5,22 +5,24 @@
   "license": "BSD-3-Clause",
   "repository": "https://github.com/GPII/gpii-binder",
   "scripts": {
-    "pretest": "node node_modules/rimraf/bin.js coverage instrumented",
-    "test": "node node_modules/testem/testem.js ci --skip PhantomJS,Chromium,Safari"
+    "instrument": "node node_modules/istanbul/lib/cli.js instrument --no-compact --output instrumented src",
+    "pretest": "node node_modules/rimraf/bin.js coverage instrumented && npm run instrument",
+    "test": "node node_modules/testem/testem.js ci --skip PhantomJS,Chromium,Safari",
```

Comment:
Safari has had intermittent errors loading testem itself, once we have OS X available as a QI target I will revisit in more depth.  Chromium is probably safe now, but needs to be confirmed in QI.